ios/RCTWebRTC/RTCVideoViewManager.m

```diff
@@ -263,11 +267,41 @@ - (void)setVideoTrack:(RTCVideoTrack *)videoTrack {
     // of its videoTrack only while this view resides in a window.
     if (videoTrack && self.window) {
         dispatch_async(_module.workerQueue, ^{
-            [videoTrack addRenderer:self.videoView];
+            [videoTrack addRenderer:self];
         });
     }
   }
 }
+#pragma mark - RTCVideoRenderer methods
+
+/**
+ * Renders a specific video frame. Delegates to the subview of this instance
+ * which implements the actual {@link RTCVideoRenderer}.
+ *
+ * @param frame The video frame to render.
+ */
+- (void)renderFrame:(RTCVideoFrame *)frame {
```

Comment:
I'd like to see this logic in the VideoTrackAdapter instead, since it already counts video frames.

Reply:
I tried to do this, and I couldn't find how to connect `onFirstFrame` in `RTCVideoViewManager` with `VideoTrackAdapter`. Can you please give me a hint on how I can do it? I have no idea how those 2 classes are connected.